.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,16 +1,15 @@
 ### Description of Change ###
 
-Describe your changes here. 
+<!-- Describe your changes here. -->
 
 ### Bugs Fixed ###
+<!-- Provide links to issues here. Ensure that a GitHub issue was created for your feature or bug fix before sending PR. -->
 
 - Fixes #
 
-Provide links to issues here. Ensure that a GitHub issue was created for your feature or bug fix before sending PR.
-
 ### API Changes ###
 
-List all API changes here (or just put None), example:
+<!-- List all API changes here (or just put None), example:
```

Comment:
One remark: "lazy" contributors might update the existing sample, in which case it won't show up.

Might be useful to add: copy the sample below or remove the <!-- and -->